tools/python/convert_onnx_models_to_ort.py

```diff
@@ -128,17 +125,35 @@ def parse_args():
                         help='Add operator specific type information to the configuration file to potentially reduce '
                              'the types supported by individual operator implementations.')
 
-    parser.add_argument('model_path', help='Provide path to directory containing ONNX model/s to convert. '
-                                           'Files with .onnx extension will be processed.')
+    parser.add_argument('--custom_op_library', type=pathlib.Path, default=None,
```

Comment:
do we need to add a test for exercising this path ?

Reply:
Added